torch/onnx/_onnx_supported_ops.py

```diff
@@ -53,11 +52,11 @@ def is_backward(self) -> bool:
         return "backward" in self.name
 
 
-def _all_aten_forward_schemas():
-    """Creates a list of _TorchSchema for all aten schemas."""
+def _all_forward_schemas():
```

Comment:
Do we need to update phrasing on the documentation page?

Reply:
I updated the doc in #84496. These changes are only so that the doc gen process does not break. Lmk if you want me to simplify changes to this file further in this PR

Reply:
sgtm, I just want to check with you if the original doc has words specifically for like supported "aten" ops or so.

Reply:
👍